extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -97,26 +126,35 @@ export class AmpVizVega extends AMP.BaseElement {
     dev().assert(!this.src_ != !this.inlineData_);
 
     if (this.inlineData_) {
+      let err;
       this.data_ = tryParseJson(this.inlineData_, err => {
+        err = err;
+      });
+      if (err) {
```

Comment:
removed. Had forgotten that tryParseJson's callback was sync, thought assert() would be throwing exception out of band